kornia/geometry/transform/crop/crop2d.py

```diff
@@ -237,8 +235,35 @@ def crop_by_boxes(tensor: torch.Tensor, src_box: torch.Tensor, dst_box: torch.Te
     h_out: int = int(bbox[0][0].item())
     w_out: int = int(bbox[1][0].item())
 
+    return crop_by_transform_mat(
+        tensor, dst_trans_src, (h_out, w_out),
+        mode=mode, padding_mode=padding_mode, align_corners=align_corners)
+
+
```

Comment:
I don’t see why this is needed unless we want a `crop_by_índices` but even in that case I don’t see why we don’t just directly call warp_affine